edb/schema/extensions.py

```diff
@@ -337,6 +337,20 @@ class DeleteExtension(
 
     astnode = qlast.DropExtension
 
+    def apply(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext,
+    ) -> s_schema.Schema:
+        testmode = context.testmode
+        context.testmode = True
```

Comment:
Hm. Why?

Reply:
Because create/deleting extensions does operations that aren't allowed in normal DDL. The create side already had this, but there was something on the delete side that ended up needing it too